pkg/skaffold/runner/dev.go

```diff
@@ -111,7 +91,16 @@ func (r *SkaffoldRunner) Dev(ctx context.Context, out io.Writer, artifacts []*la
 
 		if err := r.monitor.Register(
 			func() ([]string, error) { return r.Builder.DependenciesForArtifact(ctx, artifact) },
-			func(e filemon.Events) { r.changeSet.AddDirtyArtifact(artifact, e) },
```

Comment:
`AddDirtyArtifact` can be removed now, right?

Reply:
yep, thanks